src/particle.h

```diff
@@ -0,0 +1,160 @@
+#ifndef PARTICLE_H
+#define PARTICLE_H
+
+#include <cstdint>
+#include <array>
+
+#include "openmc.h"
+
+namespace openmc {
+
+//==============================================================================
+// Constants
+//==============================================================================
+
+constexpr int MAX_DELAYED_GROUPS {8};
+constexpr int MAX_SECONDARY {1000};
+constexpr int NEUTRON {1};
+constexpr int MAX_LOST_PARTICLES {10};
+constexpr double REL_MAX_LOST_PARTICLES {1.0e-6};
+
+extern "C" {
+
+  struct LocalCoord {
+    int cell;
+    int universe;
+    int lattice;
+    int lattice_x;
+    int lattice_y;
+    int lattice_z;
```

Comment:
Do you think it might be helpful to initialize these to something from `#include <limits>` so we hopefully see exceptions earlier if they are ever used un-initialized?